meta-lmp-bsp/recipes-security/optee/optee-os-fio_3.10.0.bbappend

```diff
@@ -34,29 +47,19 @@ EXTRA_OEMAKE_append_imx7ulpea-ucom = " \
     CFG_TZDRAM_START=0x9e000000 CFG_OVERLAY_ADDR=0x65000000 \
     CFG_OVERLAY_RESERVED_MEMORY_ADDRESS_CELLS=1 CFG_OVERLAY_RESERVED_MEMORY_SIZE_CELLS=1 \
 "
-EXTRA_OEMAKE_append_imx8mq-evk = " \
-    CFG_NXP_CAAM=y CFG_RNG_PTA=y \
-    CFG_DT=y CFG_EXTERNAL_DTB_OVERLAY=y CFG_DT_ADDR=0x43200000 \
+EXTRA_OEMAKE_append_imx8qm-mek = " \
+    CFG_NXP_CAAM=n CFG_RNG_PTA=y \
+    CFG_DT=y CFG_EXTERNAL_DTB_OVERLAY=y CFG_DT_ADDR=0x83200000 \
```

Comment:
And here.

Reply:
The only generic SoC settings are for `mx8m` which won't apply to apalis-imx8 (imx8qm)